backend/src/app/config.clj

```diff
@@ -58,11 +58,8 @@
    :srepl-host "127.0.0.1"
    :srepl-port 6062
 
-   :storage-backend :fs
-
-   :storage-fs-directory "assets"
-   :storage-s3-region :eu-central-1
-   :storage-s3-bucket "penpot-devenv-assets-pre"
+   :assets-storage-backend :fs
+   :storage-assets-fs-directory "assets"
```

Comment:
I would prefer to use always "assets-storage" or always "storage-assets", to make clear these settings are closely related.

Reply:
I think I prefer "storage-assets". This way they will be also near "storage-fdata".